scripts/build/compile

```diff
@@ -353,6 +353,9 @@ def mark(should, f):
   elif ("/backend/" in f) and ("/tests/" in f):
     should.backend_test = True
 
+  elif ("/packages/" in f) and (f.endswith(".dark")):
+    should.backend_quick_build = True
```

Comment:
What's the goal here?

Reply:
parse and re-load packages upon save of a package

might be worth
- extracting "load packages" from the run-backend-server script
- add a `should.reload_packages`
- when doing any backend build, set that^ to `true`
?

Reply:
Yeah, thats a better (and faster) approach